### ISSUE TITLE
yarle won't uninstall in Windows 10

### ISSUE THREAD

**Issue by ercbk** [akosbalasko/yarle#605]
Was going to try to update yarle, but I couldn't uninstall my current version. Every time I try to uninstall I get the message that yarle is running and I need to stop it before I can uninstall. I didn't have it open and I couldn't find the process running. I restarted Windows in safe mode, but the same problem persisted. Unless you have some other ideas, I guess the only way to remove it is by deleting the files manually. If that's the case, I was hoping you could tell me where all the files I need to delete are located.
Thank you

yarle 5.1.2
Windows 10 Pro, version 22H2, build 19045.4046

**Comment by ercbk** [comment]
I already tried that. There was no process with the name "yarle" in it.

**Comment by akosbalasko** [comment]
that's strange because I tested it too on Windows, and I have it. something like electron or npx?

**Comment by ercbk** [comment]
Are you saying your able to reproduce the issue? And if you're asking whether I have a process with something like electron or npx in the name, then I do not.

**Comment by ercbk** [comment]
I've seen the name of the yarle process when I'm using it. If that's the process that the uninstaller thinks is running, it isn't running.

**Comment by akosbalasko** [comment]
nope nope, I cannot reproduce the issue, I have the process in the Task Manager on Windows.

**Comment by akosbalasko** [comment]
getting back to the original problem: Yarle is being installed by executing the itself, it unpacks and installs all the dependencies and everyting under of a folder (I don't exactly know the folder name, probably yarle-evernote-to-md) within %AppData% . Could you pls check that folder ?

**Comment by ercbk** [comment]
Yeah it's appdata/roaming

**Comment by akosbalasko** [comment]
Yes, it should work.

**Comment by akosbalasko** [comment]
@ercbk  any news?

**Comment by ercbk** [comment]
I got it uninstalled. I ended up noticing that the folders I removed were only around 150MB (cache and installer) when Apps & Features said the program size was around 465MB. So I hadn't deleted the main program. Found the main program and deleted it. Went back to Apps & Features, Windows said that it couldn't find it and must be uninstalled. Then, I installed the latest version, and it works. So all is well. Thank you.

Locations
- AppData\local\programs # main program
- AppData\Roaming\yarle-evernote-to-md # caching
- AppData\Local\yarle-evernote-to-md-updater # installer

Shortcuts (just for completeness)
AppData\Roaming\Microsoft\Windows\Start Menu\Programs
ProgramData\Microsoft\Windows\Start Menu\Programs
